examples/js/renderers/CSS3DRenderer.js

```diff
@@ -233,27 +235,39 @@ THREE.CSS3DRenderer = function () {
 
 	}();
 
-	function zOrder( scene ) {
+	function filterAndFlatten( scene ) {
 
-		var order = Object.keys( cache.objects ).sort( function ( a, b ) {
```

Comment:
This is only for IE11.
Cached `Object.keys` will not work anymore, since it cached in a WeakMap. To make an array, I add a function called `filterAndFlatten`